README.md

```diff
@@ -31,31 +30,36 @@ Make sure you have at least 1.5 GB free disk space.
     sudo apt install libssl-dev libpq-dev libc6-dev curl gcc pkg-config
     ```
 
-3.  *ff-node-monitor* is written in [Rust](https://www.rust-lang.org/) using
+3.  Fetch the sources:
+    
+    ```
+    sudo -u ff-node-monitor git clone https://github.com/freifunk-saar/ff-node-monitor.git /opt/ff-node-monitor/src
+    ```
+
+4.  *ff-node-monitor* is written in [Rust](https://www.rust-lang.org/) using
     [Rocket](https://rocket.rs/), which means it needs a nightly version of Rust:
 
     ```
+    cd /opt/ff-node-monitor
     curl https://sh.rustup.rs -sSf > rustup.sh
-    sudo -u ff-node-monitor sh rustup.sh --default-toolchain $(cat rust-version)
+    sudo -u ff-node-monitor sh rustup.sh --default-toolchain $(cat src/rust-version)
```

Comment:
This is really odd and not obvious at all though, you previous referred to this directory as `/opt/ff-node-monitor/src` and now you use the relative path.

Reply:
But I change directly in line 37 : cd /opt/ff-node-monitor
so it is clear, that all in this **step 4.** has this directory as base.

My idea was mainly, that each single step is logical in itself, in case you don't do all steps directly after-another.